var/spack/repos/builtin/packages/openfst/package.py

```diff
@@ -36,14 +36,14 @@ class Openfst(AutotoolsPackage):
     list_url = "http://www.openfst.org/twiki/bin/view/FST/FstDownload"
 
     version('1.6.1',  '1173066ed987072183b950b54bdc9315')
-    version('1.6.0',  '2b7bcfde6b629857dc5f9ad7edd6ece9')
-    version('1.5.4',  'e52bd59ec9d9edde0c1268a382662741')
-    version('1.5.3',  'f04e580b5bc574571854304c9656a3e2')
-    version('1.5.2',  'e9d43874f7cadf791394caab3925eee4')
-    version('1.5.1',  '8869e44c5a4af65409ae78b9f482b40e')
-    version('1.5.0',  'a24fee5ffe28744c6fb7b1a49e0006c4')
     version('1.4.1',  'ca8f1730b9b9b281e515611fa9ae23c0')
-    version('1.4.0',  '662367ec91084ffab48ee9b5716de39c')
+    version('1.4.1-patch',  'ca8f1730b9b9b281e515611fa9ae23c0',
+            url='http://www.openfst.org/twiki/pub/FST/FstDownload/openfst-1.4.1.tar.gz')
```

Comment:
I deleted versions 1.5.0:1.6.0 for the following reason. `kaldi` and `cntk` seem to be the only packages depending on `openfst` by far. When I added `openfst` package, the number of available versions to include is picked quite randomly (just picked the first 10 versions I think).  Now versions `1.6.1` and `1.4.1` are the only ones that have been tested against kaldi and cntk. For simplying maintenance and user choice, I deleted other versions.

Shall I restore these deleted these versions in this case?